splice_access_api/src/main/java/com/splicemachine/access/configuration/PipelineConfiguration.java

```diff
@@ -156,6 +156,9 @@
     public static final String PIPELINE_KRYO_POOL_SIZE= "splice.writer.kryoPoolSize";
     private static final int DEFAULT_PIPELINE_KRYO_POOL_SIZE=1024;
 
+    public static final String BULK_IMPORT_SAMPLE_FRACTION = "splice.bulkImport.sample.fraction";
+    private static final double DEFAULT_BULK_IMPORT_SAMPLE_FRACTION = 0.01d;
```

Comment:
How is the sample percentage decided? What's the general guideline for changing it for varying sizes of datasets?